support-frontend/assets/pages/new-contributions-landing/components/ContributionThankYou/ContributionThankYou.jsx

```diff
@@ -64,7 +63,6 @@ function ContributionThankYou(props: PropTypes) {
           </section>
         ) : null}
         <MarketingConsent />
-        <ContributionSurvey contributionType={props.contributionType} />
```

Comment:
A thought: rather than removing any code in this PR, could we just allow `ContributionSurvey` to decide whether or not to render?
E.g. in its `render` function: `if (surveyLink) return ( <div>...</div>)`
This seems to be something we keep switching on/off

Reply:
@tomrf1 I started to implement this approach, but I think it's a little bit confusing to show the `<ContributionSurvey>` component on the thank you pages with the expectation that it won't show up. I'd rather take it off the pages and reintroduce it when we expect it to show up (and have the process of reintroducing it well documented). But I do take your point and I'm not sure how often this happens, so maybe it would be better to leave in place if we are doing a survey more than a couple times a year.

Reply:
I'd personally prefer configurable code to commented out code - I'd be good to get thoughts of @jranks123 and @joelochlann here

Reply:
I'm leaning towards keeping it in and having it optionally on. It's something that we are always taking out and putting in and I think it's happened enough now to warrant a place in the code.

Reply:
👍

Reply:
@tomrf1 @jranks123 thanks for calling it. I created a different branch and PR with the implementation here: https://github.com/guardian/support-frontend/pull/1785